### ISSUE TITLE
Split calls in gl_translate more effectively - not all or nothing?

### ISSUE THREAD

**Issue by LukasWallrich** [ropensci/googleLanguageR#80]
Currently, it appears like gl_translate attempts to send everything in one call, and failing that, makes one call per vector element. In my use case, that results in 20,000 calls of 100 characters each, all with their own status message ... not ideal, and probably slower than a few larger calls.

Would it make sense to instead attempt to split the request into (e.g.) 2, then 4, then 8 buckets? Id' be happy to propose an alternative to the current line `Reduce(rbind, lapply(t_string, gl_translate, format = format, 
                target = target, source = source, model = model))` if something like that would seem useful?

**Comment by MarkEdmondson1234** [comment]
Yes any optimisation you can do will be appreciated. Is there some control by doing some text processing before the API call?  Even if its just a helper function to split into bigger batches or something. 

There is also probably a concurrency feature, so sending multiple API calls at once may be possioble - this is an approach I've looked at in googleCloudRunner
